packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -144,5 +148,17 @@ function getTarget(instance) {
     }
   }
 
+  // if a `targetObject` CP was provided, use it
+  if (target) { return target; }
+
+  // if _targetObject use it
+  if (instance._targetObject) { return instance._targetObject; }
+
   return null;
 }
+
+function _deprecateTargetObject() {
+  let message = 'Usage of `targetObject` is deprecated. Please use `target` instead.';
+  let options = { id: 'ember-runtime.using-targetObject', until: '2.17.0' };
```

Comment:
I've let this slip yet again, and I am very sorry.  Can you update this again to 3.5.0?